src/buildercore/core.py

```diff
@@ -157,16 +157,16 @@ def boto_s3_conn(region):
 
 @cached
 def connect_aws_with_pname(pname, service, with_boto3=False):
-    "convenience"
+    "convenience. returns a boto client for a service in same region as given project"
     pdata = project.project_data(pname)
     region = pdata['aws']['region']
     LOG.debug('connecting to a %s instance in region %s', pname, region)
     if with_boto3:
-        return boto3.client('rds', region)
+        return boto3.client('rds', region) # BUG: this should be `service`, right?
```

Comment:
@giorgiosironi ?

Reply:
Yes, probably it was the only value used so far